lib/cli.ex

```diff
@@ -1,8 +1,10 @@
 defmodule Onigumo.CLI do
   def main(argv) do
-    {[], [component]} = OptionParser.parse!(argv, strict: [])
+    {parsed_switches, [component]} = OptionParser.parse!(argv, strict: [working_dir: :string])
     module = Module.safe_concat("Onigumo", component)
-    root_path = File.cwd!()
-    module.main(root_path)
+
+    parsed_switches
+    |> Keyword.get(:working_dir, File.cwd!())
+    |> module.main
```

Comment:
The Elixir style guide [suggests](https://github.com/christopheradams/elixir_style_guide?tab=readme-ov-file#parentheses-pipe-operator) using parentheses in the pipe operator.

> Use parentheses for one-arity functions when using the pipe operator (`|>`).

```suggestion
    |> module.main()
```

I’d generally incline towards using them for functions without an argument, mainly for consistency. The Guide considers that [preferred](https://github.com/christopheradams/elixir_style_guide?tab=readme-ov-file#function-calls-and-parentheses), although in other examples they use them without parens a lot themselves.

> Use parentheses in function calls, especially inside a pipeline.